docs/development/releasing.rst

```diff
@@ -214,6 +214,9 @@ packages that use the full bugfix/maintenance branch approach.)
    be better off just copying-and-pasting the relevant parts of the maintenance
    branch's ``CHANGES.rst`` into master.
 
+#. Create a github milestone for the next bugfix version, move any remaining
+   issues from the version you just released, and close the milestone.
```

Comment:
Maybe mention that if releasing a major release, close the last bugfix milestone on the previous maintenance branch as well.